src/components/App.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react';
+import ToDoList from './ToDoList';
+
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      toDoTask: '',
+      removeTask: null,
+      toDos: []
+    };
+    this.handletoDoTask = this.handletoDoTask.bind(this);
+    this.handleAddTaskToList = this.handleAddTaskToList.bind(this);
+  }
+
+  handletoDoTask(event) {
+    this.setState({toDoTask: event.target.value});
+  }
+
+  handleAddTaskToList() {
+    let todo = this.state.toDoTask;
+    const toDos = this.state.toDos;
+    toDos.push(todo);
+    this.setState({ toDos, toDoTask: '' });
+  }
+
+  // handleRemoveTaskFromList() {
```

Comment:
The only thing you need to do to fix this is to have some sort of id to compare the task you want deleted with the tasks in the array. Then run the splice over where the item is found at that index